2021-09-22 21:05:13,524 INFO    MainThread:41227 [wandb_setup.py:_flush():69] setting env: {}
2021-09-22 21:05:13,525 INFO    MainThread:41227 [wandb_setup.py:_flush():69] setting login settings: {}
2021-09-22 21:05:13,525 INFO    MainThread:41227 [wandb_init.py:_log_setup():348] Logging user logs to /home/pholur/Situation_Modeling/wandb/run-20210922_210513-3sph8a9h/logs/debug.log
2021-09-22 21:05:13,525 INFO    MainThread:41227 [wandb_init.py:_log_setup():349] Logging internal logs to /home/pholur/Situation_Modeling/wandb/run-20210922_210513-3sph8a9h/logs/debug-internal.log
2021-09-22 21:05:13,525 INFO    MainThread:41227 [wandb_init.py:init():381] calling init triggers
2021-09-22 21:05:13,526 INFO    MainThread:41227 [wandb_init.py:init():388] wandb.init called with sweep_config: {}
config: {}
2021-09-22 21:05:13,526 INFO    MainThread:41227 [wandb_init.py:init():430] starting backend
2021-09-22 21:05:13,526 INFO    MainThread:41227 [backend.py:_multiprocessing_setup():72] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2021-09-22 21:05:13,530 INFO    MainThread:41227 [backend.py:ensure_launched():135] starting backend process...
2021-09-22 21:05:13,610 INFO    MainThread:41227 [backend.py:ensure_launched():140] started backend process with pid: 41754
2021-09-22 21:05:13,612 INFO    MainThread:41227 [wandb_init.py:init():435] backend started and connected
2021-09-22 21:05:13,614 INFO    MainThread:41227 [wandb_init.py:init():494] updated telemetry
2021-09-22 21:05:13,615 INFO    MainThread:41227 [wandb_init.py:init():517] communicating current version
2021-09-22 21:05:14,330 INFO    MainThread:41227 [wandb_init.py:init():522] got version response 
2021-09-22 21:05:14,331 INFO    MainThread:41227 [wandb_init.py:init():530] communicating run to backend with 30 second timeout
2021-09-22 21:05:14,472 INFO    MainThread:41227 [wandb_init.py:init():557] starting run threads in backend
2021-09-22 21:05:18,861 INFO    MainThread:41227 [wandb_run.py:_console_start():1605] atexit reg
2021-09-22 21:05:18,862 INFO    MainThread:41227 [wandb_run.py:_redirect():1479] redirect: SettingsConsole.REDIRECT
2021-09-22 21:05:18,862 INFO    MainThread:41227 [wandb_run.py:_redirect():1484] Redirecting console.
2021-09-22 21:05:18,868 INFO    MainThread:41227 [wandb_run.py:_redirect():1540] Redirects installed.
2021-09-22 21:05:18,868 INFO    MainThread:41227 [wandb_init.py:init():582] run started, returning control to user process
2021-09-22 21:10:34,179 INFO    MainThread:41227 [wandb_run.py:_atexit_cleanup():1575] got exitcode: 255
2021-09-22 21:10:34,180 INFO    MainThread:41227 [wandb_run.py:_restore():1547] restore
2021-09-22 21:10:37,047 INFO    MainThread:41227 [wandb_run.py:_wait_for_finish():1697] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 736
  total_bytes: 736
}

2021-09-22 21:10:37,375 INFO    MainThread:41227 [wandb_run.py:_wait_for_finish():1697] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 736
  total_bytes: 736
}

2021-09-22 21:10:37,493 INFO    MainThread:41227 [wandb_run.py:_wait_for_finish():1697] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 736
  total_bytes: 736
}

2021-09-22 21:10:37,702 INFO    MainThread:41227 [wandb_run.py:_wait_for_finish():1697] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 736
  total_bytes: 14417
}

2021-09-22 21:10:37,808 INFO    MainThread:41227 [wandb_run.py:_wait_for_finish():1697] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 736
  total_bytes: 14417
}

2021-09-22 21:10:37,911 INFO    MainThread:41227 [wandb_run.py:_wait_for_finish():1697] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 14417
  total_bytes: 14417
}

2021-09-22 21:10:38,013 INFO    MainThread:41227 [wandb_run.py:_wait_for_finish():1697] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 14417
  total_bytes: 14417
}

2021-09-22 21:10:38,116 INFO    MainThread:41227 [wandb_run.py:_wait_for_finish():1697] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 14417
  total_bytes: 14417
}

2021-09-22 21:10:38,218 INFO    MainThread:41227 [wandb_run.py:_wait_for_finish():1697] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 14417
  total_bytes: 14417
}

2021-09-22 21:10:38,465 INFO    MainThread:41227 [wandb_run.py:_wait_for_finish():1697] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 14417
  total_bytes: 14417
}

2021-09-22 21:10:38,739 INFO    MainThread:41227 [wandb_run.py:_wait_for_finish():1697] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 14417
  total_bytes: 14417
}
local_info {
}

2021-09-22 21:10:40,009 INFO    MainThread:41227 [wandb_run.py:_show_files():1947] logging synced files
